src/mlpack/methods/ann/layer/convolution.hpp

```diff
@@ -76,7 +77,23 @@ class Convolution
               const size_t padH = 0,
               const size_t inputWidth = 0,
               const size_t inputHeight = 0,
-              const std::string& paddingType = "None");
+              const std::string& paddingType = "none");
+
+  /**
+   * Create the Convolution object using the specified number of output maps,
+   * filter size, stride and padding parameter. The number of input maps will
+   * be inferred.
+   *
+   * @param outSize The number of output filter.
+   * @param kernelSize The height and width of the convolution window;
+   * @param strides The height and width of the stride.
+   * @param padding The type of padding ('valid', 'same', 'none').
+   *     Defaults to 'none'.
+   */
+  Convolution(const size_t filter,
```

Comment:
```suggestion
  Convolution(const size_t outSize,
```

Same issue here, whichever way you want to solve it is fine with me. :+1: